<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import Data</class>
 <widget class="QMainWindow" name="Import Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>761</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Input data</string>
    </property>
    <widget class="QTableWidget" name="dataTypeChange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>31</y>
       <width>741</width>
       <height>301</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>15</number>
     </property>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data type</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Format</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>191</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>35</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>String</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Integer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date/Time</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>191</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>35</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>String</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Integer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Date/Time</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>&lt; back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>next &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>finish</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modify data type</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
